frontend/src/routes/_protected/vehicles/index.tsx

```diff
@@ -1,35 +1,53 @@
-import { vehicleQuery } from "@/api/queries";
-import LoadingInfo from "@/components/general/error/LoadingInfo";
-import { useSuspenseQuery } from "@tanstack/react-query";
-import { createFileRoute } from "@tanstack/react-router";
-import { Suspense } from "react";
-import { ErrorBoundary } from "react-error-boundary";
+import { vehicleQuery } from '@/api/queries'
+import VehicleCard from '@/components/general/cards/VehicleCard'
+import LoadingInfo from '@/components/general/error/LoadingInfo'
+import { useSuspenseQuery } from '@tanstack/react-query'
+import { createFileRoute } from '@tanstack/react-router'
+import { Suspense } from 'react'
+import { ErrorBoundary } from 'react-error-boundary'
+import ButtonLink from '@/components/general/links/ButtonLink'
+import { Plus } from 'lucide-react'
 
-export const Route = createFileRoute("/_protected/vehicles/")({
+export const Route = createFileRoute('/_protected/vehicles/')({
   component: Vehicles,
   meta: () => [
     {
       title: 'Fahrzeuge',
       path: '/vehicles',
     },
-  ]
-});
+  ],
+})
 
 function Vehicles() {
   const { data: vehicleRes } = useSuspenseQuery(vehicleQuery())
+  vehicleRes.data?.map((vehicle, key) => console.log(vehicle))
```

Comment:
There is a `console.log`. This should be deleted if it is only used for debugging 👍